### PR TITLE
fix install instructions & docs

### PR DESCRIPTION
**Pull request by radumg**
- fixed API docs link
- fixed install instructions for Debian/Ubuntu

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -36,11 +36,17 @@ Steps:
     $ heroku open
 
 ### Deploying on Debian-based OSes (Ubuntu etc)
+Follow this if you're using DigitalOcean for example.
```
Comment (didimitrie) [inline readme.md]: if you're using digitalocean, you should go for the one click docker app, and get going w/h docker...

**Comment by didimitrie** [comment]
I'll have to pass the review on to @yorikvanhavre, he set up these instructions! All the depolyments that i did so far were okay, but i've always hacked my way through things...

File: readme.md (hunk 1 above)
Comment (radumg) [reply readme.md]: that don't work as it stands. also one-click apps are no longer supported on DigitalOcean, so you have to install docker manually.

Comment (radumg) [reply readme.md]: umh my bad, i was wrong, was looking at the Redis one which isn't supported any more and misread it's about all of them :
https://www.digitalocean.com/community/tutorials/how-to-use-the-redis-one-click-application

Comment (radumg) [reply readme.md]: yep, tested and works fine, don't know why i couldn't get it to work when i tried earlier ¯\_(ツ)_/¯

**Comment by yorikvanhavre** [comment]
I installed the "redis" package which automatically pulled "redis-server", but if redis-server suffices, should be OK. For me (debian testing), the stock nodejs/npm from the default repo worked fine, no need to install a separate version manually. Also package name here is spelled nodejs, instead of NodeJS

**Comment by radumg** [comment]
thanks @yorikvanhavre !
`redis` isn't found as a package on Ubuntu : https://askubuntu.com/questions/868848/how-to-install-redis-on-ubuntu-16-04/868862

Spelling-wise, was writing their actual names, not the package names as they were close but not quite the same in that sentence. Should be clear it's not a command i think.

Stock node version installed on Ubuntu 16.04 is v4.x, quite outdated.

Maybe there's differences between package lists maintained on Ubuntu vs Debian ? Don't know enough about this sphere, so just trying to help make it clearer for Linux n00bs (like me) 😄

**Comment by yorikvanhavre** [comment]
wow.. my stock version of nodejs here is 8.9.. indeed ubuntu 16.04 is two years old, I guess NodeJS didn't get much attention :) Maybe we can turn this into a kind of sidenote?

**Comment by radumg** [comment]
that could work. ideally, we find instructions that work with both Debian & Ubuntu ?

**Comment by yorikvanhavre** [comment]
Yes I believe all the rest is compatible between both... Maybe something like this:

If your version of NodeJS is < 8.5 (for ex. on Ubuntu 16.04), you need to [manually install a more recent version](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-16-04), running each line below separately :

       curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
       nvm install 8.5.0

**Comment by didimitrie** [comment]
closing, as install instructions have moved on.

**Pull request closed without merging**